modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -177,8 +178,14 @@ private void runInitScriptIfRequired(final ConnectionUrl connectionUrl, Database
         if (connectionUrl.getInitScriptPath().isPresent()) {
             String initScriptPath = connectionUrl.getInitScriptPath().get();
             try {
-                URL resource = Thread.currentThread().getContextClassLoader().getResource(initScriptPath);
-
+                URL resource;
+                if (initScriptPath.startsWith(FILE_PATH_PREFIX)) {
+                    //relative project path
```

Comment:
```suggestion
                    //relative workdir path
```

"project path" (the root of Git repo? Or module? Or something else?) is too specific, I suggest using the JVM's "workdir" here, and keep it up to the user to decide what is the workdir of his tests